pages/blog/new-asyncapi-tools-page.md

```diff
@@ -0,0 +1,136 @@
+
+## Tool File Structure
+
+Here's the sample `.asyncapi-tool` file structure, which can be used to structurise your tool configuration.
+
+```JSON
+{
+  "title": "ZenWave Code Generator",
+  "description": "DDD and API-First for Event-Driven Microservices",
+  "links": {
+    "websiteUrl": "https://zenwave360.github.io/",
+    "docsUrl": "https://zenwave360.github.io/zenwave-code-generator/plugins/asyncapi-spring-cloud-streams3/",
+    "repoUrl": "https://github.com/zenwave360/zenwave-code-generator"
+  },
+  "filters": {
+    "language": "Java",
+    "technology": [
+      "Maven",
+      "Spring Cloud Streams"
+    ],
+    "categories": [
+      "code-generator",
+      "dsl",
+      "mocking-and-testing",
+      "cli"
+    ],
+    "hasCommercial": false
+  }
+}
+```
+
+This file structure should be used to insert your tool into the website. The fields specified above are explained below:
+
+- **`title`** - Specifies the title or name of the Tool. Remember this name will be used as the official name of your tool on the website.
+- **`description`** - Specify the tool description. It should be precise, up to 30 words only.
+- **`links`** - Object which contains important links related to the tool.
+  - **`websiteUrl`** - This is an optional field that specifies the URL of the tool's website.
+  - **`docsUrl`** - This is an optional field that specifies the URL of the tool's documentation.
+  - **`repoUrl`** - This is an optional field that specifies the URL of the tool's GitHub repository.
+- **`filters`** - Object that contains various fields like language, technologies, and categories to provide information about the tool.
+  - **`language`** - Specifies the primary Language in which the tool has been created. There are predefined languages listed in our documentation that are available right now and will be expanded to add new languages if needed.
+  - **`technology`** - Specifies the list of technologies that are used to create the tool. There are predefined technologies listed in our documentation that are available right now and will be expanded to add new languages if needed.
+  - **`categories`** - Specifies the list of categories that defines the type of tool. There are predefined categories listed in our documentation that can be used to list down your tool under the proper category.
+  - **`hasCommercial`** - Specifies whether the tool is a commercial product or is open source.
+
+The predefined list of technologies can be found in our repository and are listed as:
```

Comment:
The problem is that if I will specify the JSON schema along with line number, it will be not be maintainable as schema may change in future and the line numbers will differ. So, it's better that if we have the file specified for languages, technologies and categories that are being used in script, we should specify this in the blog too. WDYT?

Reply:
imho you do not have to add link to specific line. JSON Schema is not that big, it is easy to find related info.

I just think it is easier to always send back to json schema, but I will not argue about it for sure 😆

Reply:
yup